docs/guides/edge.md

```diff
@@ -67,3 +67,132 @@ with Diagram(name="Advanced Web Service with On-Premise (colored)", show=False):
 ```
 
 ![advanced web service with on-premise diagram colored](/img/advanced_web_service_with_on-premise_colored.png)
+
+## Less Edges
+
+As you can see on the previous graph the edges can become quickly noisy, here one example to solve this problem.
+
+One hack is to get creative with Custom to create blank placeholders, together with named nodes within Clusters, and then only pointing to single named elements within those Clusters.
+
+Compare the output below to the example output linked to above .
+
+```python
+from diagrams import Cluster, Diagram
+from diagrams.onprem.analytics import Spark
+from diagrams.onprem.compute import Server
+from diagrams.onprem.database import PostgreSQL
+from diagrams.onprem.inmemory import Redis
+from diagrams.onprem.aggregator import Fluentd
+from diagrams.onprem.monitoring import Grafana, Prometheus
+from diagrams.onprem.network import Nginx
+from diagrams.onprem.queue import Kafka
+from diagrams.custom import Custom
+
+with Diagram("\nAdvanced Web Service with On-Premise Less edges", show=False) as diag:
+    ingress = Nginx("ingress")
+
+    with Cluster("Service Cluster"):
+            serv1 = Server("grpc1")
+            serv2 = Server("grpc2")
+            serv3 = Server("grpc3")
+
+    with Cluster(""):
+        blankHA = Custom("","tranparent.png")
+
+        metrics = Prometheus("metric")
+        metrics << Grafana("monitoring")
+
+        aggregator = Fluentd("logging")
+        blankHA >> aggregator >> Kafka("stream") >> Spark("analytics")
+
+        with Cluster("Database HA"):
+            master = PostgreSQL("users")
+            master - PostgreSQL("slave") << metrics
+            blankHA >> master
+
+        with Cluster("Sessions HA"):
+            master = Redis("session")
+            master - Redis("replica") << metrics
+            blankHA >> master
+
+    ingress >> serv2 >> blankHA
+
+diag
+```
+
+![advanced web service with on-premise less edges](/img/advanced_web_service_with_on-premise_less_edges.png)
+
+## Merged Edges
+
+Yet another option is to set the graph_attr dictionary key "concentrate" to "true".
+
+Note the following restrictions:
+
+1.  the Edge must end at the same headport
+2.  this only works when the minlen of the Edges to be greater than "1".
```

Comment:
~~2.  this only works when the minlen of the Edges to be greater than "1".~~


*Note to Gabriel: This can work with minlen equal to 1, but in some cases it can get fussy and not layout the way you want*